build_bundle_image.sh

```diff
@@ -148,9 +148,10 @@ function check_usage {
 		echo "    LIFERAY_DOCKER_LICENSE_API_HEADER (required for DXP): API header used to generate the trial license"
 		echo "    LIFERAY_DOCKER_LICENSE_API_URL (required for DXP): API URL to generate the trial license"
 		echo "    LIFERAY_DOCKER_RELEASE_FILE_URL (required): URL to a Liferay bundle"
+		echo "    LIFERAY_DOCKER_RELEASE_VERSION (required): Version of a Liferay release"
```

Comment:
@Wykenz, is it really required or was it necessary just for your local tests?

Reply:
@natocesarrego It is required to mention every input variable in the check usage and if you meant that this variable is "(required)"  then yes, if we are not setting anything then jdk11-jdk8 will be used in any version we only set with the url

Reply:
Thanks for your clarification, @Wykenz. :slightly_smiling_face: 

The command that is currently executed to generate the Docker images is `LIFERAY_DOCKER_IMAGE_FILTER="2024.q2.0" ./build_all_images.sh`, considering that the intention is generate a Docker image associated to the release `2024.q2.0`, for example. Based on it, I realized that the variable `LIFERAY_DOCKER_RELEASE_VERSION` is being set with `2024.q2.0`, please see the following codes:

- https://github.com/liferay/liferay-docker/blob/a5813aea073d3c90d7db2add8fda6a69c10c7719/build_all_images.sh#L145
- https://github.com/liferay/liferay-docker/blob/a5813aea073d3c90d7db2add8fda6a69c10c7719/build_all_images.sh#L89

In light of this, I'm assuming that adding the variable `LIFERAY_DOCKER_RELEASE_VERSION` isn't mandatory. Does it make sense to you?

Reply:
@natocesarrego I'm not sure what you mean by that, as I can see ${version} is a local variable, but as I mentioned earlier I could cut out the version from LIFERAY_DOCKER_RELEASE_FILE_URL which is a required variable and use it only locally where the version is needed. That way we don't need the input info and if the LIFERAY_DOCKER_RELEASE_VERSION is not set, we can still have it's value. Is that sounds good to you? Or can you give me more detail on your previous comment? 
This change is only affecting the single use of build_bundle_image.sh since the  LIFERAY_DOCKER_RELEASE_VERSION is set with the build_all_images.sh